### DISCUSSION TITLE
Nacht der Untöten: Revamp

### DISCUSSION THREAD

**Discussion by mapkidd** [nzp-team/nzportable#1023] [Map Releases]
### ---> [**DOWNLOAD**](https://archive.org/download/ndu-revamp/NDU%20REVAMP.zip) <---

instructions to install:
1. extract the zip (if you haven't already)
2. drop the files in ''(where you have NZP installed)\nzp\maps\''
OR drop the ''maps'' folder in ''(where you have NZP installed)\nzp\'' (MAKE SURE YOU DON'T REPLACE)
3. boot up NZP, head to custom maps, and load ''Nacht der Untöten: Revamped''
4. enjoy!

features:
- classic perks
- new area where the old grenade wallbuy was
- teddy easter egg
- unlockable PaP (via a teleporter in the new area)
- new balcony where the BAR wallbuy was

photos:
![image](https://github.com/user-attachments/assets/6dc6f556-92a4-4161-ac9a-1d719f73b554)

![image](https://github.com/user-attachments/assets/1990a2af-9938-4b89-b79a-b899a82e03b1)

![image](https://github.com/user-attachments/assets/a99dfd9f-e2ef-43bf-9000-2d04c6bdfeec)

![image](https://github.com/user-attachments/assets/221df8fa-83cc-4bd3-aca6-0a961132493a)

**Comment by Bonniegamer202** [comment]
Any photos?

**Reply by mapkidd** [reply]
added!

**Reply by mmkkiinn** [reply]
Can you create a psp version

**Reply by mapkidd** [reply]
maybe later

**Comment by SlimoIsHere** [comment]
There's a lot of things that you might want to fix. I could make a list if needed. Sorry if I sound like a critic.

**Reply by mapkidd** [reply]
go ahead, i'm not a good mapper

**Reply by SlimoIsHere** [reply]
- You can walk through the bars next to power, all bookshelves (except for one that's blocking the way to the new area, you can't buy it either), the mystery box, and the table next to the mystery box.

- There's no model for the debris in the spawn room, and in the mystery box room.

- The bottles around Double Tap are Speed Cola bottles, and the bottles by Speed Cola are Deadshot Daquiri bottles.

**Reply by mapkidd** [reply]
i forgot to add collision in some areas, yea
i couldn't figure out how to add the debris models, will do later
couldn't figure out which was which in-editor so i'll try changing it later

**Comment by 26lake** [comment]
Got PAP at round 2 lol, put the loose change at 100 like COD so it isn't as easy to just speedrun it.

**Comment by irandiaz** [comment]
The PSP version is now available, I'm waiting for more xD
